user-guide/docs/usecases/overview.md

```diff
@@ -24,6 +24,7 @@ To help users fully embrace DesignSafe functionalities, we have developed a suit
 * [**Site Response Analysis and Model Calibration**](arduino/usecase.md) (OpenSees, SimCenter quoFEM, Jupyter, HPC)
 * [**Launching a Matlab script from a Jupyter notebook**](arduino/usecase_matlab.md) (Matlab, Jupyter, HPC)
 * [**Sensitivity Analysis, Bayesian Calibration and Forward Propagation of Uncertainties Using quoFEM**](arduino/usecase_quoFEM.md) (SimCenter quoFEM, Jupyter, HPC)
+* [**OpenSees Model Calibration**](#OpenSees Model Calibration) (Jupyter)
```

Comment:
> [!NOTE]
> This PR/branch was out of date with `main`. I merged `main` and resolved conflicts for you.

> [!WARNING]
> With this change you introduce a new broken link. Reference the (fixed) links above.

To what page on https://www.designsafe-ci.org/user-guide/ do you want to link?

1. Put the path of the URL of that page in the `()`.
2. In path replace `/` with `.md`.
3. To link to a specific heading, then append `#appropriate-lowrcase-dashed-heading-id` to path.

<details><summary>Examples</summary>

| For | Use |
| - | - |
| https://www.designsafe-ci.org/user-guide/tools/simulation/opensees/opensees/ | `[**OpenSees Model Calibration**](../tools/simulation/opensees/opensees.md)` |
| http://localhost:8000/user-guide/usecases/vantassel_and_zhang/usecase/ | `[**OpenSees Model Calibration**](vantassel_and_zhang/usecase.md)` |
| https://www.designsafe-ci.org/user-guide/usecases/vantassel_and_zhang/usecase/#overview-of-ml-examples | `[**OpenSees Model Calibration**](vantassel_and_zhang/usecase.md)`

</details>

Reply:
It was only a test and yes this was a broken link that was already there.